md/release-notes.md

```diff
@@ -214,14 +214,32 @@ Debug, the option to run a diagram without its connectors, is not supported anym
 * BS-19105	Studio Import: unable to recognize git repositories after studio migration
 * BS-19158	Default SAML configuration doesn't work with AD FS
 
+### Fixes in Bonita 7.8.3 (2019-03-07)
+#### Fixes in Studio component
+BST-49	Allow condition on a sequence flow that connect a BPMN element to a parallel gateway	
+BST-77	Diagram overview broken after pool sizing in the Studio	
+BST-94	Bonita la builder doesn't work if a process contains a decision table	
+BST-96 Impossible to import a BOS renamed in 7.8.0 in other Community Studios
+BST-100 Project explorer makes Studio considerably slower on Windows	
+
+#### Fixes in UI Designer component
+UID-11	Bonita API resource Template path truncated when exported in page.properties
```

Comment:
Missing: https://bonitasoft.atlassian.net/browse/UID-8

Reply:
it already appears in Release note 7.8.2 so I guess the query is wrong on this one

Reply:
It was called "BS-19257 Wrong place holder in the Date Picker says "mm/dd/yyyy" instead of "dd MMM yyy""